src/http/server/handlers/static_file_handler.cr

```diff
@@ -64,6 +64,14 @@ class HTTP::StaticFileHandler
       context.response.content_type = "text/html"
       directory_listing(context.response, request_path, file_path)
     elsif is_file
+      last_modified = File.stat(file_path).mtime.to_s("%a, %-d %h %C%y %T GMT")
```

Comment:
`HTTP.rfc1123_date` also ensures that the time actually *is* UTC/GMT instead of just labeling the timezone "GMT".